frontend/src/ui/TopBar.tsx

```diff
@@ -21,8 +21,8 @@ const TopBar = () => {
           justifyContent: 'space-between',
         }}
       >
-        <ShadowButton label={'About'} />
-        <ShadowButton label={'Contact'} />
+        <Link to="/about">About</Link>
+        <Link to="/contact">Contact</Link>
```

Comment:
Can you try wrapping the ShadowButton components in these links? The goal is to have the appearance of the ShadowButton and the navigation behavior of the Link